tests/loops/optimization/test_optimizer_loop.py

```diff
@@ -11,17 +11,22 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+
```

Comment:
nit: empty newlines

```suggestion
```

Reply:
hmm, sometimes we can't rely on black?

Reply:
I don't think black or isort have any rules about that